### PR TITLE
Allow for command line polling.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
This was a thing I started writing while being load shed & forgot about. Finished by adding some tests now.

**Comment by smn** [comment]
@miltontony @Rizziepit ready for review

File: unicore/distribute/scripts.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        for repo in get_repositories(repo_dir):
+            self.pull_repo(env, repo)
+
+        # http://docs.pylonsproject.org/projects/pyramid/en/1.3-branch/narr/commandline.html#cleanup # noqa
+        env['closer']()
+
+    def pull_repo(self, env, repo):
+        sm = StorageManager(repo)
+        remote = repo.remotes[0]
+        head = repo.heads[0]
```
Comment (Rizziepit) [inline unicore/distribute/scripts.py]: Is `repo.heads[0]` the same as `repo.active_branch`?

Comment (smn) [inline unicore/distribute/scripts.py]: Looks like, going with that rather.

File: unicore/distribute/scripts.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        self.notify = env['registry'].notify
+        for repo in get_repositories(repo_dir):
+            self.pull_repo(env, repo)
+
+        # http://docs.pylonsproject.org/projects/pyramid/en/1.3-branch/narr/commandline.html#cleanup # noqa
+        env['closer']()
+
+    def pull_repo(self, env, repo):
+        sm = StorageManager(repo)
+        remote = repo.remotes[0]
```
Comment (Rizziepit) [inline unicore/distribute/scripts.py]: At risk of being a pain, is there a way to get the upstream for the active branch explicitly?

Comment (smn) [inline unicore/distribute/scripts.py]: Good question. Let me check.

**Comment by Rizziepit** [comment]
+1

**Pull request merged**